client/my-sites/site-settings/media-settings/index.jsx

```diff
@@ -0,0 +1,114 @@
+		this.setState( updated );
+		this.props.markChanged();
+	}
+
+	render() {
+		const props = this.props;
+		return (
+			<Card className="media-settings site-settings">
+				<FormFieldset>
+					<div className="media-settings__info-link-container">
+						<InfoPopover position={ 'left' }>
+							<ExternalLink target="_blank" icon={ true } href={ 'https://jetpack.com/support/photon' } >
+								{ props.translate( 'Learn more about Photon' ) }
+							</ExternalLink>
+						</InfoPopover>
+					</div>
+					<JetpackModuleToggle
+						siteId={ props.siteId }
+						moduleSlug="photon"
+						label={ props.translate( 'Speed up your images and photos with Photon.' ) }
+						description="Enabling Photon is required to use Tiled Galleries."
+						disabled={ props.isRequestingSettings || props.isSavingSettings }
+						/>
+				</FormFieldset>
+				<FormFieldset className="media-settings__formfieldset has-divider is-top-only">
+					<div className="media-settings__info-link-container">
+						<InfoPopover position={ 'left' }>
+							<ExternalLink target="_blank" icon={ true } href={ 'https://jetpack.com/support/carousel' } >
+								{ props.translate( 'Learn more about Carousel' ) }
+							</ExternalLink>
+						</InfoPopover>
+					</div>
+					<JetpackModuleToggle
+						siteId={ props.siteId }
+						moduleSlug="carousel"
+						label={ props.translate( 'Transform standard image galleries into full-screen slideshows.' ) }
+						disabled={ props.isRequestingSettings || props.isSavingSettings }
+						/>
+					<div className="media-settings__module-settings is-indented">
+						<FormLabel>
+							<FormCheckbox
+								name="carousel_display_exif"
+								checked={ props.fields.carousel_display_exif || false }
+								onChange={ props.handleToggle( 'carousel_display_exif' ) }
+								disabled={ props.isRequestingSettings || props.isSavingSettings || ! props.carouselActive } />
+							<span>{ props.translate( 'Show photo metadata in carousel, when available.' ) }</span>
+						</FormLabel>
+						<FormLabel htmlFor="carousel_background_color">
+							{ props.translate( 'Background color' ) }
+						</FormLabel>
+						<FormSelect
+							name="carousel_background_color"
+							id="carousel_background_color"
+							value={ props.fields.carousel_background_color || '' }
+							onChange={ props.onChangeField( 'carousel_background_color' ) }
+							disabled={ props.isRequestingSettings || props.isSavingSettings || ! props.carouselActive } >
+							<option value={ 'black' } key={ 'carousel_background_color_black' }>
+								{ props.translate( 'Black' ) }
+							</option>
+							<option value={ 'white' } key={ 'carousel_background_color_white' }>
+								{ props.translate( 'White' ) }
+							</option>
+						</FormSelect>
+					</div>
+				</FormFieldset>
+			</Card>
+		);
+	}
+}
+
+MediaSettings.propTypes = {
+	carouselActive: PropTypes.bool.isRequired,
+	isSavingSettings: PropTypes.bool
+};
+
+const mapStateToProps = ( state ) => {
```

Comment:
IMHO `mapStateToProps` is simple enough to be kept directly in the `connect` call without assigning it to a constant. But that's your call really 😃